models/mail/src/index.ts

```diff
@@ -82,7 +82,8 @@ function createMailViewlet (builder: Builder): void {
       configOptions: {
         hiddenKeys: ['name'],
         sortable: true
-      }
+      },
+      baseQuery: { space: undefined }
```

Comment:
Why?

Reply:
It is added to not display objects from other person spaces during loading

Reply:
Fixed